src/OAuth/AccessKey.cs

```diff
@@ -20,5 +22,12 @@ public class AccessKey
 
         [JsonProperty("jwk")]
         public JsonWebKey Jwk { set; get; }
+
+        public static AccessKey CreateFromBase64EncodedAccessKey(string encoded)
+        {
+            var accessKeyStr = Encoding.UTF8.GetString(Convert.FromBase64String(encoded));
```

Comment:
There are some error cases that could be handled better. For example if a null string is passed into this function, the error returned is 
`Error: System.ArgumentNullException: Value cannot be null. (Parameter 's').`
which is thrown from `Convert.FromBase64String`

Could follow similar pattern as seen here for handling null strings so that the parameter described in the error is the name of parameter passed into the function
https://github.com/Laserfiche/lf-api-client-core-dotnet/blob/df40225f0ec6fe6ae25cc0175c83cc3129e672ca/src/Utils/DomainUtils.cs#L39-L40